src/Sulu/Component/Contact/Tests/Unit/SmartContent/AccountDataProviderTest.php

```diff
@@ -132,7 +130,7 @@ public function testResolveDataItems($filters, $limit, $page, $pageSize, $reposi
         $this->assertEquals($items, $result->getItems());
     }
 
-    public function resourceItemsDataProvider()
+    public static function resourceItemsDataProvider(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes